app/routes.js

```diff
@@ -82,6 +82,12 @@ export const routes = (authProvider) => {
     return res.render('pages/adduser/template', context);
   }));
 
+  router.get('/organisations/:organisationId/:userId', authProvider.authorise(), withCatch(authProvider, async (req, res) => {
+    const { organisationId, userId } = req.params;
+    logger.info(`navigating to organisation: ${organisationId} edit user: ${userId} page`);
+    res.send('edit user page');
```

Comment:
I know it's just temp. But it will be the view user page right?

Reply:
yep but I'm not changing it here - I have already started the next ticket!